sdk/servicebus/Microsoft.Azure.ServiceBus/tests/ServiceBusConnectionStringBuilderTests.cs

```diff
@@ -191,5 +191,15 @@ public async Task NonAmqpUriSchemesShouldWorkAsExpected()
                 await receiver.CloseAsync();
             });
         }
+
+        [Fact]
+        public void InvalidAadConnectionStringTest()
+        {
+            var connectionString = "Endpoint=sb://test.servicebus.windows.net/;authentication=Managed Identity;SHAREDACCESSKEYNAME=val";
```

Comment:
Can we please move these to `[InlineData]` and try to keep the test scope focused?   These are really different cases for the same behavior, not a related set of assertions to validate a single operation.